podman_compose.py

```diff
@@ -89,6 +89,13 @@ def parse_short_mount(mount_str, basedir):
         mount_type = "bind"
         # TODO: should we use os.path.realpath(basedir)?
         mount_src = os.path.join(basedir, os.path.expanduser(mount_src))
+        if compose.compatibility_mode:
+            try:
+                os.makedirs(mount_src, exist_ok=True)
```

Comment:
just a nitpick with this: if the bind source is an existing file and not a directory, os.makedirs will fail even with exist_ok, so a previous check is required

Reply:
nm I'm blind it's just below, deleting this comment